blueocean-dashboard/src/main/less/connection-lost.less

```diff
@@ -0,0 +1,30 @@
+.errorscreen {
+    &.connection-lost {
+        background-color: @errorscreen-bg-color;
+    }
+    &.connection-ok {
+        background-color: @blueocean-blue;
+    }
+
+    &.connection-lost, &.connection-ok {
+        /*
+         * Lets ensure that the lost and ok messages are the same
+         * width Vs jumpy message box resizes.
+         */
+        .message-box {
+            min-width: 30em;
+            max-width: 30em;
+        }
+
+        /*
+         * For some reason, we needed to set the font-weight on the
+         * h3 used in the errorscreen when disconnected. If not, it
+         * appears more bold than it does if the same screen is
+         * shown when connected, plus when compared to the 404 screen.
+         * Weird :(
+         */
+        h3 {
+            font-weight: 400;
+        }
```

Comment:
That's the problem ... for whatever reason, the browser is deciding to render one of the fallback fonts when there's no connection ....

![screenshot 2017-01-23 13 01 30](https://cloud.githubusercontent.com/assets/429311/22204887/521f936a-e16c-11e6-99ab-03e7b0e1e52a.png)